app/services/submissions/search_service.rb

```diff
@@ -1,6 +1,17 @@
 module Submissions
   class SearchService
-    SORTABLE_COLUMNS = %w[laa_reference firm_name client_name last_updated status_with_assignment risk_level service_name].freeze
+    SORTABLE_COLUMNS = %w[
```

Comment:
Doesn't directly relate to this ticket but having these fields in the common gem will help us with the future query param validation ticket, agreed?

Reply:
Agreed. I think there's a question of how much search-related stuff goes into the common gem, because a lot of this stuff knows to a certain extent about the structure of the NSM and PA payloads, but trying to move e.g. view definitions into the gem is gonna send us loopy...

Reply:
Yeah I think a sensible drawing of that line is views, and any code that is identical in both (hence, _common_). If we start going down the route of adding too much conditional logic to the gem it ends up being a dumping ground rather than a neat way to make things more DRY